### PR TITLE
Update decryption of webapp's secrets.py

### PR DESCRIPTION
**Pull request by aag**
This commit makes several changes to the decryption of webapp's secrets.py:
- Changes the password filename to the one in the latest aws-config scripts
- Changes the name of the encrypted file to the current one in the webapp repo
- Changes the openssl parameters to work with the current encrypted file in the webapp repo

Issue: https://khanacademy.atlassian.net/browse/TP-7594

### CODE REVIEW

**Review by csilvers — APPROVED** [review]
Looks good!  Glad you only have to change this in one place.  As I mentioned in the other PR, you'll want to wait until tomorrow to land this, so the workers have picked up the new image with secrets.py.aes.password in t.

**Review by yogieric — APPROVED** [review]

**Pull request merged**
